OrbitCore/LinuxUtils.h

```diff
@@ -14,12 +14,19 @@
 
 #include "OrbitBase/Result.h"
 #include "module.pb.h"
+#include "tracepoint.pb.h"
+
+#define K_LINUX_TRACING_EVENTS_PATH "/sys/kernel/debug/tracing/events"
```

Comment:
If there isn't a good reason to have this as a macro in the header I would prefer  
constexpr char kLinuxTracingEvents[] = "/sys...";
in the cpp file.